client.py

```diff
@@ -1,53 +1,67 @@
-#
-#   Request-reply client in Python
-#   Connects REQ socket to tcp://localhost:5559
-#   Sends "Hello" to server, expects "World" back
-#
 import tqdm as tqdm
 import zmq
 import numpy as np
 from utils import send_array
 from numpy import random
 
 
+class Client:
+    def __init__(self, addr="tcp://localhost:5559", port=8192):
```

Comment:
The `8192` number is not a port, but the size of the high water mark. I think that we can remove it for now and just use the default.